packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -461,9 +506,31 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
     }
 
     /**************************************
-     *        INTERNAL FUNCTIONS          *
+     *    INTERNAL & PRIVATE FUNCTIONS    *
      **************************************/
 
+    // Update the internal fee counter metrics by adding in any accumulated fees from the last time this logic was
+    // called. Considers any allocated fees that are assigned to the LPs at the conclusion of a bridging action.
+    function updateFeeCounters(uint256 allocatedLpFees) internal {
+        // Calculate the unallocatedAccumulatedFees from the last time the contract was called.
+        uint256 unallocatedAccumulatedFees = getUnallocatedAccumulatedFees();
+
+        // Decrement the undistributedLpFees by the amount of accumulated fees.
+        undistributedLpFees = undistributedLpFees > unallocatedAccumulatedFees
+            ? undistributedLpFees - unallocatedAccumulatedFees
+            : 0;
```

Comment:
Is the 0 case possible? Don't we take a min in `getUnallocatedAccumulatedFees`?

Reply:
yup! you totally right on that. This little bit of logic was a holdout from the previous implementation I believe. I've removed this operator and all works fine in the tests & I don't think there is a situation where this becomes problematic due to the `min` taken in the previous method.